tests/Integration/Requests/PrepareTest.php

```diff
@@ -73,7 +73,12 @@ public function testExecuteUpdateUpdatesRows() : void
         $insertResult1 = $stmt->executeUpdate(1, 'foo');
         $insertResult2 = $stmt->executeUpdate([':name' => 'bar'], [':id' => 2]);
 
-        $selectResult = $this->client->executeQuery('SELECT * FROM prepare_execute ORDER BY id');
+        if (server_version_at_least('2.11', $this->client)) {
```

Comment:
Btw, it seems like this test works by coincidence, and only because you run the whole test suite, which has `Requests/ExecuteTest.php`, which in turn includes `server_version_at_least()` before running this test.

Reply:
I've tried to put an independent require here, but then it all felt apart. (This is my first time with PHP). Nonetheless, I got rid of `require`.